app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -308,6 +299,7 @@ class SuggestedEditsTasksFragment : Fragment() {
     }
 
     private fun showOneTimeSequentialUserStatsTooltips() {
+        suggestedEditsScrollView.fullScroll(View.FOCUS_UP);
```

Comment:
Please remove the `;`.

Minor question: would it be possible to show the tooltips after the view is scrolled to the top?

Reply:
This line scrolls it to the top @cooltey ..